recipes/terraform-provider-keycloak/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set provider = "keycloak" %}
+{% set goname = "github.com/mrparkers/terraform-provider-"+provider.lower() %}
+{% set version = "1.17.1" %}
+
+{% set name = goname.split('/')[-1] %}
+{% set pkg_src = ('src/'+goname).replace("/",os.sep) %}
+
+{% set x4_plugin = name + '_v' + version + "_x4" + (".exe" if win else "")%}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  - folder: {{ pkg_src }}
+    url: "https://{{ goname }}/archive/{{ version }}.tar.gz"
+    sha256: 4847deaf10c845d46d11d7360ff43d7e418d3d93b5be6ea695a6672654fc98a7
+
+build:
+  number: 0
+  script:
+    - pushd {{ pkg_src }}
+    - go build -v -o $GOBIN/{{ x4_plugin }} .  # [unix]
+    - go build -v -o %GOBIN%\{{ x4_plugin }} .  # [win]
+
+requirements:
+  build:
+    - {{ compiler('go') }}
+  run:
+    - terraform ==0.11.14
```

Comment:
Does this need to be so tightly pinned?

Reply:
@scopatz thanks, just fixed this

Reply:
@scopatz please merge when you have a chance. thanks!